src/rockstor/fs/btrfs.py

```diff
@@ -127,7 +127,7 @@ def pool_raid(mnt_pt):
         if (len(fields) > 1):
             block = fields[0][:-1].lower()
             raid = fields[1][:-1].lower()
-            if not block in raid_d and raid is not 'DUP':
+            if block not in raid_d and raid is not 'DUP':
```

Comment:
Wasn't me :wink: my editor does automatic PEP8 checking, it complains about a lot of small things.